js/foundation.responsiveToggle.js

```diff
@@ -60,7 +77,7 @@ ResponsiveToggle.prototype._events = function() {
 
   $(window).on('changed.zf.mediaquery', this._update.bind(this));
 
-  this.$toggler.on('click.zf.responsiveToggle', this.toggleMenu.bind(this));
+  this.$toggler.on('click.zf.responsiveToggle', this.options.animate ? this.toggleMenuAnimated.bind(this) : this.toggleMenu.bind(this));
```

Comment:
I'd rather have this decision live within toggleMenu.  As it is, we end up with duplicated logic between toggleMenuAnimated and toggleMenu.  Lets instead have the inner part of toggleMenu check for this.options.animate and decide how to do the toggle.